src/documents/tasks.py

```diff
@@ -62,6 +70,109 @@ def train_classifier():
         logger.warning("Classifier error: " + str(e))
 
 
+def barcode_reader(image) -> list:
+    """
+    Read any barcodes contained in image
+    Returns a list containing all found barcodes
+    """
+    barcodes = []
+    # Decode the barcode image
+    detected_barcodes = pyzbar.decode(image)
+
+    if not detected_barcodes:
+        logger.debug("No barcode detected")
+    else:
+        # Traverse through all the detected barcodes in image
+        for barcode in detected_barcodes:
+            if barcode.data != "":
```

Comment:
I just filtered for the existence of the barcode. Maybe in the future we can add more filtering, but right now, I don't think its necessary.

Reply:
Since these are inserted by a scanner (right?) the quality of detection should already be quite high.  so I don't think we need to check for the quality either, for now